srcpkgs/PolyMC/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'PolyMC'
+pkgname=PolyMC
+version=1.0.6
+revision=1
+_nbtversion="dc72a20b7efd304d12af2025223fad07b4b78464"
+_quazipversion="c9ef32de19bceb58d236f5c22382698deaec69fd"
+build_style=cmake
+configure_args="-DLauncher_BUILD_PLATFORM=Void -DLauncher_LAYOUT=lin-system"
+hostmakedepends="openjdk11 qt5-host-tools qt5-qmake"
```

Comment:
Would it be unreasonable to bump this to `openjdk17`?

Reply:
their build instructions say jdk17: https://github.com/PolyMC/PolyMC/blob/develop/BUILD.md

Reply:
It builds fine with any jdk from 8-17. Are there any advantages to building with openjdk17?

Reply:
Any JDK from 8 through 17 will be fine, as the resulting JAR file will target Java 8 anyway.

Reply:
Alright, will keep it at 11 for now as long as `java-environment` points to 11.

Reply:
`java-environment` actually points to `openjdk8`